### ISSUE TITLE
how to understand "supervise" in t-SNE embedding?

### ISSUE THREAD

**Issue by MathildaHwang** [lanpa/tensorboardX#442]
when I set supervise = 0, my dataset does not cluster well. But when supervise = 5, my dataset was perfectly divided by labels.  Can it means model trained with this dataset will overfit to the training data?

**Comment by lanpa** [comment]
https://www.ibm.com/blogs/research/2017/11/interactive-supervision-tensorboard/
I think this is a feature that lets a user assign label to some of the points. Whether the data is overfitted or not, it should improve the visualization result. Btw, which kind task are you training on? If you visualize the raw mnist dataset without training in tensorboard, you can see that they are well separated.

**Comment by iamshri8** [comment]
Hello @lanpa ,

Same question. When i have 99 x 128 dim embedding layer. I tried to visualise it using t-SNE. When I set supervise = 87, i am getting 4 different clusters exactly based on my labels, but if i set supervise to 0 i am not at all getting clusters. **So what does supervise mean**? I tried searching all over the documentation and I did not find any reference regarding the same. Could you please explain about supervise hyperparameter? Is supervise mean that the clustering is done based on the labels? if so, then what is the need for clustering labels based on labels? 
In the attachment below, for the same embedding i have ran t-SNE using different values of supervise [0, 24, 87] and the screenshot is showing the results. What do I infer from that? 

![supervise=0](https://user-images.githubusercontent.com/31486526/92390007-68ec9400-f11a-11ea-8376-1e1856df54a8.png)
![supervie=24](https://user-images.githubusercontent.com/31486526/92390008-69852a80-f11a-11ea-9867-3268a828c252.png)
!supervie=87](https://user-images.githubusercontent.com/31486526/92390009-69852a80-f11a-11ea-85b4-e9eebb56b1ab.png)

Thanks for your reply in advance!
